pages/_document.js

```diff
@@ -11,6 +11,9 @@ class MyDocument extends Document {
       <Html>
         <Head>
           <meta charSet="UTF-8" />
+          <meta property="og:title" content="Members Page:RDS" />
+          <meta property="og:image" content="/images/Real-Dev-Squad@1x.png" />
```

Comment:
Is this the RDS Logo? we do not want the RDS logo to be shown as Preview. @ankushdharkar has the preview image for `members`

Reply:
I went with my instincts haha...once the image is provided I can make the change easily